### DISCUSSION TITLE
Add an option for editor verify tab to auto-refresh when a change is made

### DISCUSSION THREAD

**Discussion by DarkbitNetwork** [ppy/osu#25365] [Ideas] [answered]
Currently, the editor's verify tab only updates when you press the refresh button which isn't very easy to find*. Adding this as an option would make it quicker to fix problems that are detected by the verify tab because I don't have to remember that I already fixed this problem or having to constantly press refresh button every time I make a change to the beatmap.

*: _This might be a "me problem" but at the time, I just assumed that I had to re-enter the editor each time to update the verify tab until I figured out that refresh button exists which took me time to find its location, it would be nicer if it's moved to somewhere more visible like the tab on the right._

**Comment by Gabixel** [comment]
That's probably the ultimate goal. I remember [this being discussed](https://github.com/ppy/osu/pull/12361#issuecomment-817141228) when someone first made the AiMod (the verify thing)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/25376
